transcendence-app/src/auth/auth.module.ts

```diff
@@ -0,0 +1,7 @@
+import { Module } from '@nestjs/common';
+import { FortyTwoModule } from './fortytwo/fortytwo.module';
+
+@Module({
+  imports: [FortyTwoModule],
+})
+export class AuthModule {}
```

Comment:
Why do we need this module? Why not having just 42 auth module for now?

Reply:
I guess we don't need it. I thought its a good idea to have an auth module, and inside that module, import other types of authentication modules.

```
./auth
  ./oauth42
  ./local (email/username and password)
  ./github
  ./etc.
```